src/server_manager/messages/en.json

```diff
@@ -97,7 +97,7 @@
   "gcp-firewall-create-3": "Type '0.0.0.0/0' in the 'Source IP ranges' field.",
   "gcp-firewall-create-4": "Select 'Allow all' under 'Protocols and ports'.",
   "gcp-firewall-create-5": "Click 'Create'.",
-  "key": "Key",
+  "key": "Key {keyId}",
```

Comment:
@alalamav Do we need to update the messages in all languages before releasing?

Reply:
Yes, this will break non-English displayed names for unnamed keys to just say whatever the word is for "key"

Reply:
It would break translations. We have two options: wait for the new string to come out the tranlsation pipeline before importing it, or updating the string for all languages now.

Reply:
Addressed, see my below comment.  This PR is now a no-op until we send the strings out for translation